gapis/service/service.proto

```diff
@@ -731,6 +733,8 @@ service Gapid {
       returns (stream GetTimestampsResponse) {
   }
 
+  // ValidateDevice runs validation of the GPU driver on the requested
+  // device to check that GPU counters are within an expected range.
```

Comment:
to validate the GPU profiling capabilities of the requested device

Reply:
done

Reply:
Unfortunately now the line is too long :)

Reply:
Indeed! fixed.